crypto/curve25519/curve25519.c

```diff
@@ -653,50 +653,50 @@ static void fe_invert(fe out, const fe z) {
   for (i = 1; i < 2; ++i) {
     fe_sq(t1, t1);
   }
-  fe_mul(t1, z, t1);
-  fe_mul(t0, t0, t1);
+  GFp_fe_mul(t1, z, t1);
```

Comment:
Does this change the code generation in any significant way? In other words, is the C compiler already generating regular function falls for this, or was it doing fancy inlining stuff?